test.R

```diff
@@ -0,0 +1,7 @@
+library(epinowcast)
+
+ nowcast <- enw_example("nowcast")
+ summarised_nowcast <- summary(nowcast)
+ obs <- enw_example("observations")
+
+enw_score_nowcast(summarised_nowcast, obs)
```

Comment:
What is this script for?

Reply:
ack - that is left over from making a reprex for #198 will remove.

Reply:
Removed in https://github.com/epinowcast/epinowcast/pull/200/commits/f2d6fc03895d1448749f9fec2f7a2ca4c818df35